olwebsite-app/src/components/Objects/Input/Input.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export default function Input({ className, label, type='text' }) {
```

Comment:
We might want to have this as a controlled component, especially since we'll be submitting forms.

Reply:
Right now this PR is just setting up initial look, I think we can add all the functionality when we connect it to the API